packages/mu-http/driver.js

```diff
@@ -14,4 +14,144 @@
 
 'use strict'
 
-// TODO
+var http = require('http')
+var stringify = require('fast-safe-stringify')
+var parse = require('fast-json-parse')
+var assert = require('assert')
+var mue = require('mu-error')()
+
+
+/**
+ * HTTP transport
+ * Figure out path mapping - perhaps this may need to be part of the config for now just /
+ */
+module.exports = function createTcpDriver (options) {
+  return function httpDriver (opts, receive) {
+    var connections = {}
+    var connectionsByIp = {}
+    var server
+
+    assert(opts, 'transport should always pass opts to httpDriver')
+    assert(receive instanceof Function, 'transport should always pass receive function to httpDriver')
+
+
+    function send (message, cb) {
+      var sendTo
+
+      if (connections[message.protocol.dst]) {
```

Comment:
suggest breaking the two main cases into two functions - primarily for readability - but also makes inlining more likely